src/mount/zwo.rs

```diff
@@ -174,6 +174,12 @@ impl Mount for ZWO {
             send_cmd_and_get_reply(&mut self.serial_port, ":Mgw0500#".into(), ResponseType::None).map(|_| ())?;
```

Comment:
@ddompe @Pichlerdom This (guiding pulse length in ms) should rather be 9999 instead of 0500. We don't want the mount to make a short pulse, Vidoxide itself controls that. When the program judges guiding should end, it will send another command with speed = 0.

Reply:
OK, the max by AM5 is 3000, let's change to it